samples/bluetooth/iso_combined_bis_and_cis/README.rst

```diff
@@ -0,0 +1,125 @@
+.. _bluetooth_isochronous_combined_bis_and_cis:
+
+Bluetooth: ISO Combined BIS and CIS
+#####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Bluetooth® Combined BIS and CIS sample demonstrates the capability of
+a Controller to support both Broadcast Isochronous Stream (BIS) and Connected Isochronous Stream (CIS) simultaneously.
+It shows ISO-specific APIs to create a stream and to receive and send data over either CIS or BIS.
+The sample acts as a Central device that receives data over CIS and forwards the received data to a BIS.
+
+Note: The sample uses the Time of Arrival method to send data over ISO without using timestamps through a single stream (see :ref:`iso_providing_data`).
+The :ref:`bluetooth_isochronous_time_synchronization` sample can be used to learn how to synchronize data using ISO channels.
+
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+.. include:: /includes/tfm.txt
+
+You can use any of the development kits listed above and mix different development kits.
+
+.. include:: /includes/hci_ipc_overlay.txt
+
+The sample also requires a connection to a computer with a serial terminal |ANSI| for each of the development kits.
+
+Overview
+********
+
+The sample demonstrates a device combining the CIS Central and BIS Source roles.
+As a CIS Central, it only receives data from CIS Peripheral.
+The received data is then broadcasted over a BIS.
+To provide the best performance and to minimize the number of conflicts between the CIS, BIS and ACL roles, the following measures are taken:
+* The same ISO Interval is used for both CIS and BIS. The value is set to 10 ms by default.
+* ACL Interval is set to an integer multiple of ISO Interval to avoid collisions. The value is set to 20 ms by default.
+* Periodic advertising of BIS is set to the same value as ACL Interval.
+* The number of retransmissions is set to 10, to ensure that packets are retransmitted in case of collisions.
+* The sample rejects incoming Connection Update requests from the peripheral to avoid changes to the ACL interval.
+* In addition, the following optional SDC-specific configurations are used:
+   * :kconfig:option:`CONFIG_BT_CTLR_SDC_CENTRAL_ACL_EVENT_SPACING_DEFAULT` is set to 10 ms to ensure that there is enough time available to place the periodic advertising packet between ACL packets.
+   * :kconfig:option:`CONFIG_BT_CTLR_SDC_MAX_CONN_EVENT_LEN_DEFAULT` is set to a small value 2.5 ms because the sample doesn't send any data over ACL connection (except Control procedures).
```

Comment:
```suggestion
   * :kconfig:option:`CONFIG_BT_CTLR_SDC_MAX_CONN_EVENT_LEN_DEFAULT` is set to a small value 2.5 ms because the only data the sample sends over the ACL connection is Control procedures.
```